docs/team/jiangchen.adoc

```diff
@@ -32,8 +32,8 @@ as it required changes to existing commands.
 * *Other contributions*:
 
 ** Enhancements to existing features:
-** Refactor original AddressBook and Person class into ExpenseTracker and Expense class.
-** Update sample entries.
+** Refactors original AddressBook and Person class into ExpenseTracker and Expense class.
```

Comment:
Maybe use past tense i.e Refactored/Updated? Current tense being used might be grammatically incorrect